package.json

```diff
@@ -50,7 +50,7 @@
 		"standard-version": "^9.3.2"
 	},
 	"dependencies": {
-		"@typescript-eslint/eslint-plugin": "^5.12.0",
+		"@typescript-eslint/eslint-plugin": "^5.24.0",
 		"@typescript-eslint/parser": "^5.12.0",
```

Comment:
Could you update this version to match the line above. They probably should remain fairly in sync